images/win/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -3,10 +3,9 @@ function Get-OSName {
 }
 
 function Get-OSVersion {
-    $systemInfo = Get-CimInstance -ClassName Win32_OperatingSystem
-    $OSVersion = $systemInfo.Version
-    $OSBuild = $systemInfo.BuildNumber
-    return "OS Version: $OSVersion Build $OSBuild"
+    $systemInfo = cmd /c ver
+    $OSVersion = $systemInfo[$systemInfo.Length-1]
+    return "OS Version: $OSVersion"
```

Comment:
Please return back changes with Cim Class

Reply:
Wow, that's what I get for testing locally on Win2019.  I thought QFE number has been included in `ver` for years... :innocent: